conf/test_nanopore.config

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
we should probably name the original test to `test_illumina` for accuracy

Reply:
fixed